images/fedora.yaml

```diff
@@ -36,6 +36,42 @@ targets:
           lxc.arch = {{ image.architecture_kernel }}
 
 files:
+  - path: /etc/default/grub
+    generator: dump
+    content: |-
+      # Set the recordfail timeout
+      GRUB_RECORDFAIL_TIMEOUT=0
+
+      # Do not wait on grub prompt
+      GRUB_TIMEOUT=0
+
+      # Set the default commandline
+      GRUB_CMDLINE_LINUX_DEFAULT="console=tty1 console=ttyS0"
+
+      # Set the grub console type
+      GRUB_TERMINAL=console
+
+      # Disable os-prober
+      GRUB_DISABLE_OS_PROBER=true
+    types:
+      - vm
+
+  - path: /etc/dracut.conf.d/lxd.conf
+    generator: dump
+    content: |-
+      add_drivers+=virtio_scsi
+    types:
+      - vm
+
+  - generator: fstab
+    types:
+      - vm
+
+  - path: /etc/machine-id
+    generator: dump
+    types:
+      - vm
```

Comment:
Any reason not to do it for containers too, same as we do for Debian/Ubuntu?

Reply:
Actually, given you're now always generating a machine-id as part of the image, we absolutely need to clear the file in both containers and vms, or we'll end up with all our containers having the same unique id.

Reply:
Yeah, that's wrong. I'll fix that.